erlang.mk

```diff
@@ -47,18 +47,26 @@ verbose_0 = @
 verbose_2 = set -x;
 verbose = $(verbose_$(V))
 
+ifeq ($V,3)
+SHELL := $(SHELL) -x
+endif
+
 gen_verbose_0 = @echo " GEN   " $@;
 gen_verbose_2 = set -x;
 gen_verbose = $(gen_verbose_$(V))
 
+gen_verbose_esc_0 = @echo " GEN   " $$@;
+gen_verbose_esc_2 = set -x;
+gen_verbose_esc = $(gen_verbose_esc_$(V))
+
 # Temporary files directory.
 
 ERLANG_MK_TMP ?= $(CURDIR)/.erlang.mk
 export ERLANG_MK_TMP
 
 # "erl" command.
 
-ERL = erl +A0 -noinput -boot start_clean
+ERL = erl -noinput -boot no_dot_erlang -kernel start_distribution false +P 1024 +Q 1024
```

Comment:
Does this work with the old OTP versions we claim to support?

Reply:
I tested `make`, `make tests`, `make docs` and `make dialyze` with OTP versions 21+, which is all we now support.

The reason for updating `erlang.mk` was that with the old version, `make dialyze` would not run on recent OTP versions because it was using a warning option that was removed at some point, namely `race_conditions`.